code-examples/quarkus/kafka-quickstart/.kubernetes/kubernetes.yml

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
Why this deeply nested folder structure?

Reply:
Looking at this more, we really don't need this - the sample can remain in `code-examples` for now.

Reply:
technically this is rhoas gudes - so we can host more than quarkus-kafka example. I'm trying to avoid renames in future etc. and putting right structure in place.